index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+  <section id="header">
+    <nav class="navbar">
+      <div>
+        <img src="image/My_Logo.png" alt="My Logo">
+      </div>
+      <div class="d-navbar">
+        <a class="a-navbar" href="#Portfolio">Portfolio</a>
+        <a class="a-navbar" href="#About">About</a>
+        <a class="a-navbar" href="#Contact">Contact</a>    
+      </div>
+    </nav>
+  </section>
+  
+  <section id="headline">
+        <h1 class="welcome">I am Paka <br> Glad to see you!</h1>
+
+        <p class="introduction">I’m a software developer! I can help you build a product , feature or website Look through
+           some of my work and experience! If you like what you see and have a project you need
+            coded, don’t hestiate to contact me.
+        </p>
+      <div class="link-up">
+        <h2 >LET'S CONNECT</h2>
+        <ul>
```

Comment:
Good job using a `ul` tag with `li` items. I see that you have yet to update the `a` tag `href` attribute. Please update them to their relevant links.